src/app/feature/consultant/components/consultant-skills/consultant-skills-edit/consultant-skills-edit.component.ts

```diff
@@ -40,6 +42,11 @@ export class ConsultantSkillsEditComponent implements OnInit {
 
   updateConsultant() {
     this.skillEditService.updateConsultant();
+    if (this.skillEditService.updateConsultant() == void {}) {
```

Comment:
I believe this will call update twice. The skillEditService should be reworked to return an observable but for now this should be fine. You might be able to get rid of line 44 and it might have the same effect.

Reply:
I need to correct myself. This implementation won't actually be fine. We'll need to move the notification call into the skills edit service on line 237 and 242.